src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -101,6 +101,23 @@ public void whenStreamTimeoutReachedPossibleToCommit() throws Exception {
         Assert.assertEquals(SC_NO_CONTENT, statusCode);
     }
 
+    @Test(timeout = 10000)
+    public void whenStreamTimeoutReachedThenEventsFlushed() throws Exception {
```

Comment:
I tested streaming using this same parameter in the old stack and the new one and both behave the same way. Both got 2 batches, one containing 3 events and the other with debug message. Does this test actually exercise or prove that the fix works?

Reply:
I remember that I was running this test against the old version and it failed. I will check it again now.

Reply:
Oh yes, sorry, I remembered now. I changed the test later and it actually now works for old version as well (as it reaches max_uncommitted_events)
I know changed the test. It will now fail in old version and will work for new version.

Reply:
https://github.com/zalando/nakadi/pull/860/commits/0b8a0e19138906dfe173d9af6c1babb12e872ea9